pkg/string/v1alpha1/string.go

```diff
@@ -0,0 +1,41 @@
+// Copyright © 2018-2019 The OpenEBS Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package strings
+
+import (
+	lib_strings "strings"
+)
+
+// List holds the array of strings
+type List struct {
+	items []string
+}
+
+// MakeList will make list of string slices
+func MakeList(entry ...string) *List {
+	l := &List{items: []string{}}
+	l.items = append(l.items, entry...)
+	return l
+}
+
+// Contains will return true if it has matching string
```

Comment:
is this fn intention is to find matching string or string that has given 'search' as substring?
this fn is not for matching string.. its for matching substring..

Reply:
I will update the func name as `IsSubStringPresent` or there any better suggestions?

Reply:
deleted this package